ui/page/dcrdex/dex_onboarding_page.go

```diff
@@ -494,9 +540,23 @@ func (pg *DEXOnboarding) formFooterButtons(gtx C) D {
 			if !addBackBtn {
 				return D{}
 			}
+			pg.goBackBtn.SetEnabled(backBtnEnabled)
 			return layout.Inset{Right: u10}.Layout(gtx, pg.goBackBtn.Layout)
 		}),
 		layout.Flexed(nextFlex, func(gtx C) D {
+			if pg.isLoading {
+				return layout.Flex{}.Layout(gtx,
+					layout.Flexed(1, func(gtx C) D {
+						return layout.Center.Layout(gtx, func(gtx C) D {
+							gtx.Constraints.Max.X = gtx.Dp(values.MarginPadding20)
+							gtx.Constraints.Min.X = gtx.Constraints.Max.X
+							return pg.materialLoader.Layout(gtx)
+						})
+					}),
+				)
```

Comment:
We're already inside a flex layout, is it necessary to wrap the loader inside another flex layout with flex = 1??

Reply:
Tested and it appears it is sufficient to just use layout.Center alone, without wrapping in a flex.